### PR TITLE
Feature/43273

### PR DESCRIPTION
**Pull request by Aekal**
dev-43273

### CODE REVIEW

File: styles/vendor/fotorama/modules/_gallery-placeholder.scss [hunk 1]
```diff
@@ -1,10 +1,6 @@
 .gallery-placeholder {
-    .loading-mask {
+    .loader.loader--visible {
```
Comment (szafran89) [inline styles/vendor/fotorama/modules/_gallery-placeholder.scss]: please change to `&--visible`

**Comment by szafran89** [comment]
Please update changelog as well.

File: web/mage/gallery/gallery.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            >
+                <div class="loader loader--visible">
+                    <div class="loader__circle"></div>
+                </div>
+            </div>
+            <div
+                class="fotorama__arr fotorama__arr--next fotorama__arr--disabled"
+                tabindex="-1"
+                role="button"
+                aria-label="Next"
```
[the remark this answers is not in the document]
Comment (Aekal) [reply web/mage/gallery/gallery.html]: It's default magento :P But ok, I'll add them

[the remark this answers is not in the document]
Comment (szafran89) [reply web/mage/gallery/gallery.html]: this on `data-bind="attr: {aria-label: $t('Previous')}"` should work. Are you sure you changed in correct place?

[the remark this answers is not in the document]
Comment (szafran89) [reply web/mage/gallery/gallery.html]: https://magento.stackexchange.com/questions/181070/magento-2-translate-input-placeholder-attribute-string-in-knockout-html-templat

[the remark this answers is not in the document]
Comment (Aekal) [reply web/mage/gallery/gallery.html]: no effect :|

**Pull request merged**